_scripts/instruction-widget/install.js

```diff
@@ -94,26 +96,31 @@ module.exports = function(context) {
   debian_install = function() {
     template = "debian";
 
-    if (context.distro == "ubuntu" && context.version == 16.04) {
-      context.above_4 = false;
-      context.xenial = true;
```

Comment:
We can update the template to remove all the `xenial` tags as well. Yay simplicity!